paddle/fluid/pybind/inference_api.cc

```diff
@@ -18,6 +18,7 @@
 #include <iostream>
 #include <map>
 #include <string>
+#include <unordered_set>
```

Comment:
Please add also `#include <memory>` to make cpplint check pass.

Reply:
Done